src/scaffold/entry_type/coordinator.rs

```diff
@@ -355,16 +355,7 @@ pub fn delete_handler(entry_def: &EntryDefinition) -> String {
         .collect();
 
     let delete_depending_links = match linked_from_fields.is_empty() {
-        true => format!(
```

Comment:
after lots of digging (I understand a lot more about the scaffolding codebase now), I understand why these two statements are removed here but not in the `false` arm. That said, I wonder if it could be DRYed up a bit, since those statements live in two places (here, to retrieve and delete backlinks; and in the collections file, to retrieve and delete by-author links).

Additionally,

Reply:
mmmh I would like to think so, but the way the generated code is handled is different in both the [collections](https://github.com/c12i/scaffolding/blob/4ecad64595a7b8b7313ef969d55b8939d8961e68/src/scaffold/collection/coordinator.rs#L247) and [entry-type](https://github.com/c12i/scaffolding/blob/4ecad64595a7b8b7313ef969d55b8939d8961e68/src/scaffold/entry_type/coordinator.rs#L344).

Where the latter handles the code as separate statements hence why they are inserted separately.